config/initializers/algoliasearch.rb

```diff
@@ -0,0 +1,9 @@
+AlgoliaSearch.configuration = {
+	application_id: "YE495E6TKG",
```

Comment:
Funky whitespace again here 👾 

Also a few things about these keys: 

* These values _(`application_id`, `api_key`)_ most likely shouldn't be hard-coded in this project anywhere
* You added placeholders in the `config/secrets.yml` for these, so you should use the built-in Rails secrets support here.

I'd recommend modifying the setup slightly to be something like the following:

In `config/secrets.yml`:

```ruby
algolia:
  api_key: <%= ENV.fetch("ALGOLIA_API_KEY") %>
  application_id: <%= ENV.fetch("ALGOLIA_APPLICATION_ID") %>
```

Note the `algolia` key to namespace the values. Also note the `ENV.fetch`, which will cause the application to crash on boot if the values are not available _(ensuring you don't have an environment inadvertently missing these values)_. To ease this in development, I recommend using something like [`dotenv`](https://github.com/bkeepers/dotenv) or `figaro` or similar and checking in a default `.env` file. The `.env` file doubles as documentation of what ENV values are necessary for the app to run. I like a pattern of defaulting the `.env` values to something like:

```
ALGOLIA_API_KEY=insecure-algolia_api_key
ALGOLIA_APPLICATION_ID=insecure-algolia_application_id
```

The `insecure-` prefix makes it obvious in development that these are meant to be sensitive values. The suffix matches the ENV variable name, which means that it is easy to know which value is missing if you are expecting something different. 

In the initializer here, you can then use the Rails secrets support as intended:

```ruby
AlgoliaSearch.configuration = {
  api_key: Rails.application.secrets.algolia["api_key"],
  application_id: Rails.application.secrets.algolia["application_id"],
  # ...
}
```